src/ops/ops.go

```diff
@@ -207,3 +211,48 @@ func (o *ops) renderControllerCm() error {
 	}
 	return nil
 }
+
+func (o *ops) GetVGByPV(pvName string) (string, error) {
+	output, err := o.ExecPrivilegeCommand(o.logWriter, "vgs", "--noheadings", "-o", "vg_name,pv_name")
+	if err != nil {
+		o.log.Errorf("Failed to list VGs in the system")
+		return "", err
+	}
+
+	lines := strings.Split(output, "\n")
+	for _, line := range lines {
+		res := strings.Fields(line)
+		if len(res) < 2 {
+			continue
+		}
+
+		if strings.Contains(res[1], pvName) {
+			return res[0], nil
+		}
+	}
+	return "", nil
+}
+
+func (o *ops) RemoveVG(vgName string) error {
+	output, err := o.ExecPrivilegeCommand(o.logWriter, "vgremove", vgName, "-y")
+	if err != nil {
+		o.log.Errorf("Failed to remove VG %s, output %s, error %s", vgName, output, err)
+	}
+	return err
+}
+
+func (o *ops) RemoveLV(lvName, vgName string) error {
+	output, err := o.ExecPrivilegeCommand(o.logWriter, "lvremove", fmt.Sprintf("/dev/%s/%s", vgName, lvName), "-y")
+	if err != nil {
+		o.log.Errorf("Failed to remove LVM %s, output %s, error %s", fmt.Sprintf("/dev/%s/%s", vgName, lvName), output, err)
+	}
+	return err
+}
+
+func (o *ops) RemovePV(pvName string) error {
```

Comment:
looks like this one will run only after RemoveVG , why it can't be ran from it? then it can be private

Reply:
i am using LVM conventions: creating VG will automatically create PV, but removing VG will not remove PV. Don't want any  implicit issue that somebody will be surprised with

Reply:
so why not to delete PV inside delete vg

Reply:
u know that we need to delete PV if we delete VG so they always come together, i think much better to have single flow, removeVG and do there what u want